Cargo.toml

```diff
@@ -9,6 +9,7 @@ members = [
   "crypto/dleq",
   "crypto/frost",
 
+  "coins/ethereum",
```

Comment:
We show precedence to Ethereum over Monero? We really don't care about privacy :(

ACK. No issue here ranking by significance.